src/Cache/Cache.php

```diff
@@ -168,7 +168,18 @@ protected static function _buildEngine($name)
         }
 
         $config = static::$_config[$name];
-        $registry->load($name, $config);
+
+        try {
+            $registry->load($name, $config);
+        } catch (RuntimeException $e) {
+            if (!array_key_exists('fallback', $config)) {
```

Comment:
Depending on how long the cache engine is down and how many requests are received in the mean time, is there a risk of potentially filling up the logs quite quickly by logging here?